### PR TITLE
Feature/wishlist

### PR DESCRIPTION
**Pull request by SHIKHASINGH1506**
1. Added wish list management 
  1.1 From the navbar, I can navigate to my wish list 
  1.2 From product card add/remove item from wish list
  1.3 From my wish list page move item to cart or remove from wish list
  1.4 if item is already in cart then move to bag from my wish list will increase the quantity in cart

### CODE REVIEW

File: src/components/Products/ProductCardView/ProductCard.jsx [hunk 1]
```diff
@@ -17,18 +18,30 @@ const ProductCard = ({product}) => {
...
         token 
             ? isProductInCart
                 ? navigate('/Cart') 
-                : addProductToCart(dispatch, {product: products}, token)
+                : addProductToCart(dispatch, {product: product}, token)
             : navigate('/Login');
     }
+
+    const addToWishlistHandler = (product) => {
+        console.log(product);
```
Comment (rutvikpumak) [inline src/components/Products/ProductCardView/ProductCard.jsx]: remove console

File: src/services/wishlistService.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+const token = JSON.parse(localStorage.getItem("token"));
+
+const addProductToWishlist = async (dispatch, data) => {
+  const Headers = {authorization: token};
+  const {data: {wishlist}} = await axios.post(wishlistBaseUrl, data, {headers: Headers});
+  dispatch({type: 'UPDATE_WISHLIST', payload: wishlist});
+}
+
+const deleteProductFromWishlist = async (dispatch, id) => {
+  const Headers = {authorization: token};
```
Comment (rutvikpumak) [inline src/services/wishlistService.js:14]: use template literal

**Review by rutvikpumak — APPROVED** [review]
Suggested some changes and you are good to go after that.

Code is well refactored now and looks clean as well

**Comment by tushargoyal812** [comment]
1.code is very clean and good.
2.liked how you structured files and folders.
3.code is to the point and variable naming is good.

**Pull request merged**